packages/server-boilerplate/src/microService/auth/middleware.ts

```diff
@@ -65,15 +61,21 @@ export const buildBasicBearerAuthMiddleware = (
       );
     }
 
+    let userObject: UserType;
     let accessPolicyObject: AccessPolicyObject;
     if (authHeader.startsWith('Bearer')) {
-      accessPolicyObject = await getBearerAccessPolicy(authenticator, authHeader);
+      const { user, accessPolicy } = await getBearerAccessPolicy(authenticator, authHeader);
+      userObject = user;
+      accessPolicyObject = accessPolicy;
     } else if (authHeader.startsWith('Basic')) {
-      accessPolicyObject = await getBasicAccessPolicy(authenticator, authHeader, apiName);
+      const { user, accessPolicy } = await getBasicAccessPolicy(authenticator, authHeader, apiName);
+      userObject = user;
+      accessPolicyObject = accessPolicy;
     } else {
       throw new UnauthenticatedError('Could not authenticate');
     }
 
+    req.user = userObject;
```

Comment:
The changes to attach the userObject here weren't actually needed in the end for this feature, as we're using a custom auth middleware. But regardless, I felt it was worthwhile to keep the changes as having access to the request's user object can be quite useful!